protocol/messenger_sync_activity_center_test.go

```diff
@@ -0,0 +1,121 @@
+	s.NoError(err)
+	s.Equal(1, int(num))
+	num, err = theirMessenger.persistence.SaveActivityCenterNotification(notification, true)
+	s.NoError(err)
+	s.Equal(1, int(num))
+
+	// check existence of notification in both messengers
+	notificationByID, err := s.m.persistence.GetActivityCenterNotificationByID(id)
+	s.NoError(err)
+	s.Equal(notification, notificationByID)
+	s.True(notificationByID.Read)
+
+	notificationByID, err = theirMessenger.persistence.GetActivityCenterNotificationByID(id)
+	s.NoError(err)
+	s.Equal(notification, notificationByID)
+	s.True(notificationByID.Read)
+
+	// sync unread
+	_, err = s.m.MarkActivityCenterNotificationsUnread(context.Background(), []types.HexBytes{id}, now+1, true)
+	s.NoError(err)
+
+	_, err = WaitOnMessengerResponse(theirMessenger, func(r *MessengerResponse) bool {
+		return r.ActivityCenterState() != nil
+	}, "activity center notification state not received")
+	s.NoError(err)
+	notificationByID, err = theirMessenger.persistence.GetActivityCenterNotificationByID(id)
+	s.NoError(err)
+	s.False(notificationByID.Read)
+
+	s.NoError(theirMessenger.Shutdown())
+}
+
+func (s *MessengerSyncActivityCenterSuite) TestSyncDeleted() {
+	theirMessenger, err := newMessengerWithKey(s.shh, s.privateKey, s.logger, nil)
+	s.NoError(err)
+	PairDevices(&s.Suite, theirMessenger, s.m)
+
+	now := uint64(time.Now().Unix())
+	id := types.HexBytes{0x01}
+	notification := &ActivityCenterNotification{
+		ID:               id,
+		Timestamp:        now,
+		Type:             ActivityCenterNotificationTypeMention,
+		Read:             true,
+		Dismissed:        false,
+		Accepted:         false,
+		MembershipStatus: ActivityCenterMembershipStatusIdle,
+		Deleted:          false,
+		UpdatedAt:        now,
+	}
+	num, err := s.m.persistence.SaveActivityCenterNotification(notification, true)
+	s.NoError(err)
+	s.Equal(1, int(num))
+	num, err = theirMessenger.persistence.SaveActivityCenterNotification(notification, true)
+	s.NoError(err)
+	s.Equal(1, int(num))
+
+	// check existence of notification in both messengers
+	notificationByID, err := s.m.persistence.GetActivityCenterNotificationByID(id)
+	s.NoError(err)
+	s.Equal(notification, notificationByID)
+	s.False(notificationByID.Deleted)
+
+	notificationByID, err = theirMessenger.persistence.GetActivityCenterNotificationByID(id)
+	s.NoError(err)
+	s.Equal(notification, notificationByID)
+	s.False(notificationByID.Deleted)
+
+	// sync deleted
+	_, err = s.m.MarkActivityCenterNotificationsDeleted(context.Background(), []types.HexBytes{id}, now+1, true)
+	s.NoError(err)
+
+	_, err = WaitOnMessengerResponse(theirMessenger, func(r *MessengerResponse) bool {
+		return r.ActivityCenterState() != nil
+	}, "activity center notification state not received")
+	s.NoError(err)
+	notificationByID, err = theirMessenger.persistence.GetActivityCenterNotificationByID(id)
+	s.NoError(err)
+	s.True(notificationByID.Deleted)
+
+	s.NoError(theirMessenger.Shutdown())
+}
```

Comment:
I think there're more actions on notifications available?
Would be cool to have tests for them as well. Probably our comments above about missing syncing will be reproduced.